### PR TITLE
Selenium 4

### PR DESCRIPTION
**Pull request by jowerner**

### CODE REVIEW

**Review by h-arlt — COMMENTED** [review]
Looks good to me.

Regarding the changes to the API signature file, I wasn't able to run the SigTest tool w/ Java11 so I could not verify these changes.

**Comment by jowerner** [comment]
> Regarding the changes to the API signature file, I wasn't able to run the SigTest tool w/ Java11 so I could not verify these changes.

Tracked as separate issue in #197.

File: samples/testsuite-xlt/src/scripting/testcases/assertNotVisibleScriptDeveloperOnly.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+package scripting.testcases;
+import org.junit.Ignore;
+import org.junit.Test;
+import com.xceptance.xlt.api.engine.scripting.AbstractWebDriverScriptTestCase;
+import scripting.modules.Open_ExamplePage;
+
+/**
+ * <p>This test case should have no Java wrapper class, since the functionality seems not to be supported by HTMLUnit.</p>
+ */
+@Ignore
```
[the remark this answers is not in the document]
Comment (jowerner) [reply samples/testsuite-xlt/src/scripting/testcases/assertNotVisibleScriptDeveloperOnly.java:25]: This test case had no java export file until I reexported the XML scripts. The test case does indeed not work with XltDriver, so I ignored it again right away. Looks as if off-screen elements are nevertheless visible with XltDriver. Have to check who is right. Tracked as separate issue #198.

[the remark this answers is not in the document]
Comment (h-arlt) [reply samples/testsuite-xlt/src/scripting/testcases/assertNotVisibleScriptDeveloperOnly.java:25]: Roger that. Maybe we should rename it to something like `assertNotVisibleRealBrowserOnly`. WDYT?

[the remark this answers is not in the document]
Comment (jowerner) [reply samples/testsuite-xlt/src/scripting/testcases/assertNotVisibleScriptDeveloperOnly.java]: Done.

**Review by h-arlt — APPROVED** [review]

**Pull request merged**